airflow/www/extensions/init_appbuilder.py

```diff
@@ -563,6 +575,11 @@ def security_converge(self, dry=False) -> dict:
         """
         return self.sm.security_converge(self.baseviews, self.menu, dry)
 
+    def get_url_for_login_with(self, next_url: str | None = None) -> str:
```

Comment:
This was added recently in the FAB's app builder and is used in some scenarios, so I added it too.